app/models/spree/printables/invoice/base_view.rb

```diff
@@ -66,6 +66,18 @@ def after_save_actions
 
     private
 
+    def formatted_number
+      if (Object.const_get('::Spree::PrintInvoice::NumberFormatter') rescue false)
```

Comment:
Avoid using `rescue` in its modifier form.

Reply:
Or better use defined?(::Spree::PrintInvoice::NumberFormatter). So you do not need a rescue.